packages/beacon-node/test/spec/utils/specTestIterator.ts

```diff
@@ -66,7 +66,7 @@ export const defaultSkipOpts: SkipOpts = {
     /^capella\/light_client\/single_merkle_proof\/BeaconBlockBody.*/,
     /^deneb\/light_client\/single_merkle_proof\/BeaconBlockBody.*/,
   ],
-  skippedTests: [],
+  skippedTests: [/incorrect_not_enough_consolidation_churn_available/],
```

Comment:
add a TODO-electra here

Reply:
Talked to Hsiao-wei about this test and we are looking to remove/rewrite it in the next spec test release. Will share more context on this